menu.html

```diff
@@ -53,84 +53,47 @@ <h1>Cooks!!!</h1>
+                       </div>
+                      <div class="card-content">
+                        <p>
+                          Japanese food is my forte,pick one of my main dishes, you will not be disappointed!
+                        </p>
                      
-                    </div>
-                    <div class="card-content">
-                      Hi there, the name is Mushu! I'm Mulan's personal chef but I'm
-             here to share my homecooking skills with those who are craving
-             delish Japanese food. I've got Sushi, Tempura, and Ramen. What's
-             your pick?
-                    </div>
+                      </div>
+                      </a>
                   </div>
                 </div>
                 <div class="col s12 m4">
-                  <div class="card  z-depth-4" id="chefUrsula">
-                    <div class="card-image">
-                      <img src="assets/images/ursula.jpeg" alt="">
-                     
-                    </div>
-                    <div class="card-content">
-                      Hello! My name is Ursula, and I reign the sea! In my free time,
-             other than stealing people's voices, I love to cook! Seafood is my forte,
-            pick one of my main dishes, you will not be disappointed!
-                    </div>
+                   <div class="card  z-depth-4" id="chefUrsula">
+                    <a href="cookPages/chefUrsula/ursula.html" target="_blank">
+                      <div class="card-image">
+                        <img src="assets/images/ursula.jpeg" alt="">
+                      </div>
+                      <div class="card-content">
+                        <p>
+                          Seafood is my forte,pick one of my main dishes, you will not be disappointed!
+                        </p>
+                      
+                      </div>
+                     </a>
                   </div>
                 </div>
               </div>
             </div>
-            <div class="card-content">
-              Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nostrum ea deserunt officia, dicta sint
-              perferendis.
-            </div>
-          </div>
-        </div>
-        <div class="col s12 m4">
-          <div class="card  z-depth-4" id="chefUrsula">
-            <div class="card-image">
-              <img src="assets/images/food4.jpeg" alt="">
-
-            </div>
-            <div class="card-content">
-              Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nostrum ea deserunt officia, dicta sint
-              perferendis.
-            </div>
-          </div>
-        </div>
-      </div>
-    </div>
-  </section>
-  <footer class="page-footer #b71c1c red darken-4">
-    <div class="footer-copyright ">
-      <div class="container">
-        © 2020 Copyright
-      </div>
-    </div>
-  </footer>
```

Comment:
just change this to:

`<footer class="page-footer #b71c1c red darken-4">
            <div class="container">
              © 2020 Copyright 
            </div>
          </footer>`